pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -82,6 +82,9 @@ type ControllerOptions struct {
 	// XDSUpdater will push changes to the xDS server.
 	XDSUpdater model.XDSUpdater
 
+	// TrustDomain used as SNA
```

Comment:
do you plan to change ```GetIstioServiceAccounts```  to return identity format include trust_domain, as you did in other files like controller in consul/mem

Reply:
Exactly. This is already the case if you have  look at the call chain:

```GetIstioServiceAccounts``` in ```kube/controller.go```
-> ```InstancesByPort```
-> ```kubeToIstioServiceAccount```
-> ```spiffe.MustGenSpiffeURI```

```spiffe.MustGenSpiffeURI``` uses the trustDomain which was previously configured with ```spiffe.SetTrustDomain```

Reply:
I see, thanks for explaining.